recipes/atoti/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.3.0" %}
+
+package:
+  name: atoti
+  version: {{ version }}
+
+source:
+  url: https://github.com/atoti/atoti/releases/download/v{{ version }}/atoti-{{ version }}.tar.gz
+  sha256: a7c4793c1a082091c88d6ef3760f714add65ea39f1998a0fe5ce063aec3c271e
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  build:
+    - python =3.7
+    - setuptools
+  run:
+    - python >=3.7
+    - pandas
+    - pyarrow !=0.14.1
+    - pyyaml
+
+
+test:
+  requires:
+    - openjdk
+    - pandas
+
+about:
+  home: https://www.atoti.io
+  license: https://www.atoti.io/eula/
```

Comment:
This needs to be the kind of license per spdx not a link

Reply:
Hello @beckermr 
Thank you for your review and your help.
This is a custom license so I don't know which kind of SPDX I should use. Is there a "Custom" option or something ? What should I do ?

Reply:
You should use a LicenseRef expression. @isuruf can help with which one.